### PR TITLE
Query params

### PR DESCRIPTION
**Pull request by LucasZibell**
## Summary
Change how the query params are passed to the functions. Now they are passed as an object in a second parameter 

Before:
`get('tests/get/users?limit=2')`
After:
`get('tests/get/users', { limit: 2 })`

## Screenshots
image

### CODE REVIEW

**Comment by FrankIglesias** [comment]
1. Have you checked to call the limit without any attribute what is the result?
2. Why not checking if the path contains `?` formatting the following string into an object in that case?

**Comment by LucasZibell** [comment]
1. If you call `limit` without a value firebase throws an error telling you that limit needs 1 argument.
```
FirebaseError: Function Query.limit() requires 1 argument, but was called with 0 arguments.
```
2. The reason of this pr is to avoid having `?` in the paths. And use this like a normal api
```
api.get('path', body)
```

File: src/Mock/getMock.js [hunk 1]
```diff
@@ -93,7 +94,8 @@ export const getCollectionMock = {
     }
   },
   getOnlyThree: {
-    path: 'tests/get/users?limit=3',
+    path: 'tests/get/users',
+    body: { limit: 3 },
```
Comment (DLotero) [inline src/Mock/getMock.js]: don't we know what happens when something along the lines of `{limit : -1}`, `{limit : 'lalalalala'}` or some other invalid value for the limit is passed?

Comment (LucasZibell) [reply src/Mock/getMock.js]: I think firebase will throw an error. But you are right. It is a very good idea to add a test with this failure case :+1:

**Pull request merged**